### ISSUE TITLE
Add MIPS architecture disassembler

### ISSUE THREAD

**Issue by sphinxc0re** [das-labor/panopticon#269]
We should implement a disassembler for MIPS binaries. Specifically _NEC Vr 4300_ MIPS binaries since this is the processor embedded in Nintendo 64 consoles.

Datasheet: http://datasheets.chipdb.org/NEC/Vr-Series/Vr43xx/U10504EJ7V0UMJ1.pdf

**Comment by flanfly** [comment]
I want MIPS, but we still don't have a complete AMD64 disassembler and no ARM. I consider both more important. I'll merge it but I don't have time to help with the development :sob:

**Comment by m4b** [comment]
Agreed, I wouldn't mind working on the arm at some point (I mean ideally, but inreality probably don't have time for it) but I dunno how to add a new disassembler backend. I think we also need a "Howto create a new architecture backend with RREIL"

P.S. I am terrified of amd64.

**Comment by sphinxc0re** [comment]
Seems like there are many MIPS architectures:
```
mips
mips:3000
mips:3900
mips:4000
mips:4010
mips:4100
mips:4111
mips:4120
mips:4300
mips:4400
mips:4600
mips:4650
mips:5000
mips:5400
mips:5500
mips:5900
mips:6000
mips:7000
mips:8000
mips:9000
mips:10000
mips:12000
mips:14000
mips:16000
mips:16
mips:mips5
mips:isa32
mips:isa32r2
mips:isa32r3
mips:isa32r5
mips:isa64
mips:isa64r2
mips:isa64r3
mips:isa64r5
mips:sb1
mips:loongson_2e
mips:loongson_2f
mips:loongson_3a
mips:octeon
mips:octeon+
mips:octeon2
mips:xlr
mips:micromips
```
